var/spack/repos/builtin/packages/py-pil/package.py

```diff
@@ -37,7 +37,9 @@ class PyPil(Package):
 
     provides('pil')
 
-    extends('python')
+    # py-pil currently only works with Python2.
+    # If you are using Python 3, try using py-pillow instead.
+    extends('python@1.5.2:2.8')
```

Comment:
I just tried this out but there's a problem. As it is now, `extends()` can only take a package name. If it includes a version specifier as well, it will build, but `spack activate` won't work. You'll have to change this to:
```python
extends('python')
depends_on('python@1.5.2:2.8')
```